util/memory/tracker.go

```diff
@@ -317,8 +351,8 @@ func (t *Tracker) String() string {
 
 func (t *Tracker) toString(indent string, buffer *bytes.Buffer) {
 	fmt.Fprintf(buffer, "%s\"%d\"{\n", indent, t.label)
-	if t.bytesLimit > 0 {
-		fmt.Fprintf(buffer, "%s  \"quota\": %s\n", indent, t.FormatBytes(t.bytesLimit))
+	if t.bytesHardLimit > 0 {
```

Comment:
Should we add a description for soft limit setting?

Reply:
ditto.